edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Http.Test/DefaultHttpResponse.cs

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Needed because DefaultHttpResponse was part of aspnet's "pubternal" source, which they then made private. I copied the class from their source and added a link for reference.

Reply:
It looks like DefaultHttpResponse is not needed, it could just be removed from TwinControllerTest, unless it is used somewhere else?

Reply:
It is used here:
https://github.com/and-rewsmith/iotedge/blob/andsmi/dotnet-3.1-force-0/edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Http.Test/TwinsControllerTest.cs#L205